Tools/Cli-LoRa-Device-Provisioning/CommandLineOptions/AddGatewayOption.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tools.CLI.Options
+{
+    using System;
+    using CommandLine;
+
+    [Verb("add-gateway", HelpText = "Add a new gateway device to IoT Hub.")]
+    public class AddGatewayOption
+    {
+        internal const int DefaultSpiSpeed = 8;
+        internal const int DefaultSpiDev = 0;
+        internal const string DefaultAzureIotEdgeVersion = "1.4";
+        internal const string DefaultLnsHostAddress = "ws://mylns:5000";
+        internal const string DefaultNetwork = "quickstartnetwork";
+
+        [Option(
+            "reset-pin",
+            Required = true,
+            HelpText = "Pin"
+            )]
+        public int ResetPin { get; set; }
+
+        [Option(
+            "spi-speed",
+            Default = DefaultSpiSpeed,
+            Required = false,
+            HelpText = "SPI speed"
+            )]
+        public int SpiSpeed { get; set; }
+
+        [Option(
+            "spi-dev",
+            Required = false,
+            Default = DefaultSpiDev,
+            HelpText = "SPI dev"
+            )]
+        public int SpiDev { get; set; }
+
+        [Option(
+            "device-id",
+            Required = true,
+            HelpText = "Gateway device identifier"
+            )]
+        public string DeviceId { get; set; }
+
+        [Option(
+            "api-url",
+            Required = true,
+            HelpText = "API URL to LoRa Function"
+            )]
+        public Uri ApiURL { get; set; }
+
+
+        [Option(
+            "api-key",
+            Required = true,
+            HelpText = "API authorization key"
+        )]
+        public string ApiAuthCode { get; set; }
+
+        [Option(
+            "lns-host-address",
+            Required = false,
+            Default = DefaultLnsHostAddress,
+            HelpText = "LNS host address"
+            )]
+        public string TwinHostAddress { get; set; }
+
+        [Option(
+            "network",
+            Required = false,
+            Default = DefaultNetwork,
+            HelpText = "Network name"
+            )]
+        public string Network { get; set; }
```

Comment:
This network is related to the discovery, correct? Can we adapt the HelpText to make it clearer? Something like "Network identifier for LNS Discovery purposes"

Reply:
This parameter is optional, used to tag the device in IoT Hub.
Like this:
![image](https://user-images.githubusercontent.com/227765/198202430-bdb355f5-5455-4728-90e5-717f428a39b5.png)

Reply:
But now, understand. You are right. Changed made.